modules/apps/portal-configuration/portal-configuration-settings/src/main/java/com/liferay/portal/configuration/settings/internal/SettingsLocatorHelperImpl.java

```diff
@@ -582,6 +588,13 @@ public void removedBundle(
 			}
 		}
 
+		private static final List<String> _thirdPartyPIDs =
+			Collections.unmodifiableList(
+				ListUtil.fromArray(
+					"org.apache.felix.eventadmin.impl.EventAdmin",
+					"org.apache.felix.scr.ScrService",
+					"org.apache.cxf.workqueues"));
```

Comment:
Could we make this a portal property? Just in case a new lib gets added, then we wouldn't need to change the source of this class to accommodate. Thoughts?